packages/feedback/src/environments/environment.stage.ts

```diff
@@ -0,0 +1,16 @@
+// The file contents for the current environment will overwrite these during build.
+// The build system defaults to the dev environment which uses `environment.ts`, but if you do
+// `ng build --env=stage` then `environment.stage.ts` will be used instead.
+// The list of which env maps to which file can be found in `.angular-cli.json`.
+
+export const environment = {
+  production: false,
+  name: 'stage',
+  api: 'https://api.stage.one.redhat.com',
+  graphqlAPI: 'https://api.stage.one.redhat.com/graphql',
+  client: 'https://stage.one.redhat.com',
+  authIssuer: 'https://auth.stage.redhat.com/auth',
+  matomoSiteID: 9, // Site ID from Piwik Analytics
+  serviceNowURL: 'https://redhatqa.service-now.com',
+  subscription: 'wss://api.stage.one.redhat.com/subscriptions'
+};
```

Comment:
Can you please remove this hosts from here and please use the .env files for managing this information

Reply:
did it,

its only for stages and local environment or its for prod and qa as well ?

Reply:
moved to .env file.